bundles/org.openhab.binding.clementineremote/README.md

```diff
@@ -0,0 +1,337 @@
+# Clementine Remote Binding
+
+<img style="float: right; margin: 0 0 5px 5px" src="doc/screenshot.png" alt="Screenshot" />
+This binding brings the benefits of over-the-network control of the [Clementine-Player] to openHAB!
+
+It allows control of playback, position and volume and allows display of various title information:
+
+* Artist
+* Album
+* Title
+* current position within track
+* Album cover
+
+## Discovery
+
+This thing does not provide auto-discovery.
+You will have to manually configure clementine to allow remote connections.
+
+Detailed instructions how to configure Clementine's remote functions are provided here:
+
+https://github.com/clementine-player/Android-Remote/wiki/How-to-use-the-Android-Remote
+
+And you will have to manually configure the Thing instances to talk to your player:
+
+## Thing Configuration
+
+Configuring the connection to Clementine is quite easy:
+
+Normally you only need to set the hostname your Clementine player is running at.
+If you set a password in the Clementine configuration, you will have to provide it to the configuration, too.
+
+### Clementine Thing Configuration
```

Comment:
To consider: remove the whole header as it seems to be a duplicate of parent header.

Reply:
You are right. Fixed.